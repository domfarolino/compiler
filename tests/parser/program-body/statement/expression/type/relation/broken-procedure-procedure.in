program dominic is

  bool result;

  procedure f()
  begin
  end procedure;

  procedure g()
  begin
  end procedure;

begin

  result := f <= g;

end program.
