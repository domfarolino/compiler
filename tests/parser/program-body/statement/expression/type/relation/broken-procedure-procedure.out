[1;31mLine 15: [Relation]: procedure and procedure cannot be compared
Line 15: Error parsing program body
[0m